config/dev.exs

```diff
@@ -2,6 +2,11 @@ use Mix.Config
 
 config :ex_queue_bus_client,
   queue: "communication_bus_develop",
-  event_handler: ExQueueBusClient.EventHandler
+  event_handler: ExQueueBusClient.EventHandler,
+  sns_client: ExAws.SNS,
+  aws_client: ExAws
 
-import_config "dev.secret.exs"
+config :ex_aws,
+  access_key_id: [System.get_env("AWS_ACCESS_KEY"), :instance_role],
```

Comment:
Why dev.exs uses env variables like this? 
Other PR I saw it was the opposite. System.get_env() was for test and prod envs if I remember correctly

Reply:
because they are compile time accessible.